internal/pkg/cli/pipeline_init_test.go

```diff
@@ -4,10 +4,10 @@
 package cli
 
 import (
-	"fmt"
 	"testing"
 
 	climocks "github.com/aws/amazon-ecs-cli-v2/internal/pkg/cli/mocks"
+	archerMocks "github.com/aws/amazon-ecs-cli-v2/mocks"
```

Comment:
nit: @sonofachamp pointed out to me that the idiomatic way is "archermocks" (lowercase for package names)